kubernetes/services/ignite_config.xml

```diff
@@ -45,6 +45,15 @@ Copyright(c) 2020 Futurewei Cloud
 
       <property name="cacheConfiguration">
          <list>
+             <bean class="org.apache.ignite.configuration.CacheConfiguration">
+                 <!-- Set the cache name. -->
+                 <property name="name" value="com.futurewei.alcor.web.entity.vpc.VpcEntity"/>
+                 <!-- Set the cache mode. -->
+                 <property name="atomicityMode" value="TRANSACTIONAL"/>
+                 <!-- Other cache parameters. -->
+                 <property name="cacheMode" value="PARTITIONED"/>
+             </bean>
+
              <bean class="org.apache.ignite.configuration.CacheConfiguration">
                 <!-- Set the cache name. -->
                 <property name="name" value="dpm_nodeinfo_cache"/>
```

Comment:
Looks good. Best to actually try to start Ignite with this change, one can never be sure about XML nestings and Ignite's naming oddities.